examples/search_examples/public/search_sessions/app.tsx

```diff
@@ -33,7 +33,7 @@ import { catchError, map, tap } from 'rxjs/operators';
 import { lastValueFrom, of } from 'rxjs';
 
 import { CoreStart } from '@kbn/core/public';
-import { mountReactNode } from '@kbn/core/public/utils';
```

Comment:
As explained in the issue's description, all imports from `@kbn/core/public/utils` have been replaced with their internal package counterpart.